cmd/containerd-shim-wasm-v2/main.go

```diff
@@ -17,10 +17,12 @@ package main
 */
 
 import (
-	"github.com/containerd/containerd/runtime/v2/shim"
+	"context"
+
+	"github.com/containerd/containerd/v2/pkg/shim"
 	wasm "github.com/dmcgowan/containerd-wasm"
 )
 
 func main() {
-	shim.Run("io.containerd.wasm.v1", wasm.New)
+	shim.Run(context.Background(), wasm.NewManager("io.containerd.wasm.v1"))
 }
```

Comment:
@dmcgowan  Sorry I should've explicitly commented about it. It looks like the change of command name (`containerd-shim-wasm-v1` → `containerd-shim-wasm-v2`) was not needed maybe?